sidebars.js

```diff
@@ -145,9 +145,14 @@ const sidebars = {
           items: [
             {
               type: "doc",
-              label: "Node metrics",
+              label: "Setting up metrics",
```

Comment:
[question] out of curiosity, where does this label appear? I don't see it in the sidebar, page title, or page header in the PR preview